tests/test_decorators.py

```diff
@@ -550,7 +550,7 @@ def strict_fields(self, data, original_data):
                     if key not in self.fields:
                         raise ValidationError('Unknown field name', key)
 
-        schema = MySchema()
+        schema = MySchema(unknown=EXCLUDE)
```

Comment:
This test is fine for now, but it'd be good to change it soon, now that there's a built-in way to handle unknown fields.

Reply:
I'm not sure I understand.

Do you mean that it is good as a test but bad as an example, so we should change it to a more useful or at least more neutral example?

Reply:
Exactly: the `strict_fields` implementation is a bad example that should be replaced by a more neutral one.

Reply:
Done.